### DISCUSSION TITLE
How to sort sequence of files for merge in lossless cut?

### DISCUSSION THREAD

**Discussion by abrogard** [mifi/lossless-cut#2272] [Support]
I am new with lossless cut.  I have 80 mp4 files I would like to merge into one.  I think it can do it.  It took them all in with a drag and drop.  But they have got the order mixed up.  I'm prepared to sort it out manually onscreen but can't find a way.  Is there a way?

**Comment by mifi** [comment]
hi. after drag-dropping 80 files, you can drag files around in the left side batch file list to reorder
